tests/conftest.py

```diff
@@ -18,16 +22,14 @@ def pytest_configure() -> None:
     """
     setup_sentry()
 
-    # There is only one cluster in test, so fetch the host from there.
-    cluster = settings.CLUSTERS[0]
+    for cluster in settings.CLUSTERS:
```

Comment:
Is there not a single cluster in test anymore ?

Reply:
There is still, but it's unnecessary to make that assumption. If we change that in the future, I wanted this to still work.